autogpt_platform/backend/backend/data/user.py

```diff
@@ -130,3 +132,46 @@ async def migrate_and_encrypt_user_integrations():
             where={"id": user.id},
             data={"metadata": Json(raw_metadata)},
         )
+
+
+async def get_active_user_ids_in_timerange(start_time: str, end_time: str) -> list[str]:
```

Comment:
I suggest passing in the `datetime` instead of the string repr

Reply:
does not seralize correctly over pyro

Reply:
that's fixable, like we did for Pydantic models

Reply:
- #9472

Not tested, take or leave at your discretion